app/views/episodes/_form_tags.html.erb

```diff
@@ -1,15 +1,21 @@
-<div class="col-12 mb-4">
+<div class="col-12 mb-4" data-controller="category">
   <div class="card shadow border-0">
     <div class="card-header card-header-light d-flex justify-content-between">
       <h2 class="card-title h5"><%= t(".title") %></h2>
     </div>
 
     <div class="card-body">
       <div class="form-floating input-group">
-        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true} %>
+        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true, category_target: "field"} %>
         <%= form.label :categories %>
         <%= field_help_text t(".help.categories") %>
       </div>
     </div>
+
+    <div class="card-footer" data-controller="tooltip">
+      <button type="button" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-title="<%= t(".tooltip.adfree") %>" data-bs-placement="bottom" value="adfree" data-action="click->category#addCategory"><%= t(".label.adfree") %></button>
```

Comment:
so, the `badge` class on a button element is a little weird in terms of the text vs background (ends up being light on light, very unreadable) so instead i just turned it into `btn-sm` to at least make them small buttons and for the css to layer properly.

Reply:
Oh, I see.  That is a bit odd... you could toggle the `text-dark` class as well?  But even then, I can't see the difference between the badge color and the background on my monitor.  So this works!